tools/ci_build/github/azure-pipelines/linux-cpu-minimal-build-ci-pipeline.yml

```diff
@@ -177,6 +177,18 @@ jobs:
             /bin/bash /onnxruntime_src/tools/ci_build/github/linux/ort_minimal/build_minimal_ort_android_baseline_and_report_bin_size.sh
       workingDirectory: $(Build.SourcesDirectory)
 
+
+  - task: AzureCLI@2
+    displayName: 'Azure CLI'
+    condition: and (succeeded(), eq(variables['Build.SourceBranch'], 'refs/heads/master'))
```

Comment:
Can we still have ` $(Build.Reason) == "IndividualCI" || $(Build.Reason) == "BatchedCI"` here, otherwise the data will be to noisy